backend/WebApi/Controllers/RestaurantController.cs

```diff
@@ -186,5 +186,17 @@ public IActionResult GetAllFoodFromRestaurant(string id, string sortOrder = null
 
             return Ok(foods);
         }
+
+        /// <summary>
+        /// Shows if the log in credentials are correct
+        /// </summary>
+        /// <param name="creds">Credentials of the restaurant</param>
+        /// <returns></returns>
+        [HttpPost("Login", Name = nameof(Login))]
+        [ProducesResponseType((int)HttpStatusCode.OK)]
+        public IActionResult Login([FromBody] Credentials creds)
+        {
+            return Ok(_restaurantService.Login(creds));

```

Comment:
Can you send instead the id of the restaurant as we do in the registration endpoint? Without the id we don't know who logged in.

Reply:
Changed it